blazor/rich-text-editor/how-to/update-value.md

```diff
@@ -44,7 +44,7 @@ To achieve this, the `keydown` event has to be bound to the Rich Text Editor con
         "CreateLink", "Image", "|", "ClearFormat", "Print",
         "SourceCode", "FullScreen", "|", "Undo", "Redo"
     };
-    public void onKeydown(UIKeyboardEventArgs arg) {
+    public void onKeydown(KeyboardEventArgs arg) {
         if (arg.Key == "s" && arg.CtrlKey == true) {
             this.rteObj.UpdateValue();
```

Comment:
is it runnable code? have you ensured the code snippet?

Reply:
ensured in code, it is working